src/components/Select.js

```diff
@@ -212,7 +212,7 @@ const styles = {
     position: 'relative'
   },
   downArrow: {
-    color: '#999999',
+    color: '#000000',
```

Comment:
Since selectedStyle is a prop, can I access it outside the Select object, though?